### ISSUE TITLE
Feature: Sort the list of Alarm Tile on the basis of Alarm Time

### ISSUE THREAD

**Issue by superiorsd10** [CCExtractor/ultimate_alarm_clock#58]
### Description

Currently, the list of `Alarm Tile` on the `Home Page` is sorted on the basis of the latest toggled which resulted in ON. However, throughout my usage of the application, I didn't find it quite appealing and good for user experience. 

A better alternative can be to sort the list of alarms on the basis of `Alarm Time` on fetching the data from the firebase. It will enhance the user experience to a greater extent as the user will be able to navigate to their alarm more easily. 

I've thought of doing this without duplicating the list to reduce memory usage. 

If possible, please assign me this issue.

### Screenshots

https://github.com/CCExtractor/ultimate_alarm_clock/assets/92971894/4d6199a6-0b84-4167-899f-24ed6c59cd19

**Comment by MarkisDev** [comment]
The alarms are sorted by whichever is going to ring next. It doesn't make sense to show disabled alarms up top, so all the enabled alarms are shown first, but both enabled and disabled alarms are sorted by their arrival time.

**Comment by superiorsd10** [comment]
@MarkisDev Okay :)

**Comment by MarkisDev** [comment]
I've been thinking about this. How about we give user the option to enable an option to sort the alarms in ascending order?

**Comment by superiorsd10** [comment]
@MarkisDev Yes! We can give them the option on the `Settings` page. So should I work on it?

**Comment by MarkisDev** [comment]
Yes, please go ahead. I'll assign this to you.

**Comment by superiorsd10** [comment]
@MarkisDev Thank you! I'll be working on it.
